python/Circluar Linked List/LinkedList.py

```diff
@@ -0,0 +1,93 @@
+# Python program to Search an Element in a Circular Linked List
+# Md Zeeshan

```

Comment:
Explain more about the problem, with some test cases and validate those test cases with your drive code.